app/views/books/_translations.html.erb

```diff
@@ -1,4 +1,8 @@
 <p>
+This book is available in
+  <a href="/book/en">English</a>.
+</p>
+<p>
```

Comment:
Makes sense. Do wording nits:

  - we now say "this book" in two consecutive paragraphs. Should it be "This book is available in English" followed by "It has been translated into..." or something?

  - is it worth mentioning that the book isn't just available in English, but that this is the primary/original language?